### DISCUSSION TITLE
a performance question

### DISCUSSION THREAD

**Discussion by muturgan** [uNetworking/uWebSockets.js#170] [Q&A] [answered]
Well, thank you very much for this product!

I haven't found a better way to ask a question, so I'm writing this issue.

I read this article:
https://levelup.gitconnected.com/will-node-js-forever-be-the-sluggish-golang-f632130e5c7a

But I didn't really understand the comparison of nginx with application servers. Why are these tools in the same row?
Should I conclude that you recommend not proxying your uwebsockets.js application server behind the nginx and distribute static files as well via uwebsockets.js (for example, if I make an api for a spa web application)?

**Comment by ghost** [comment, marked as answer]
I don't know, you can do whatever you want to as long as you benchmark your solution for your particular use case. This lib makes sense for some special applications where it can make a big difference - other use cases maybe won't see any significant difference. Up to you to do what you find useful

**Comment by muturgan** [comment]
well thank you for the answer
